hw_diag/views/diagnostics.py

```diff
@@ -57,6 +57,7 @@ def read_diagnostics_file():
 def get_diagnostics_json():
     diagnostics = read_diagnostics_file()
     response = jsonify(diagnostics)
+    response.headers.set('X-Robots-Tag', 'none')
```

Comment:
@marvinmarnold can you set headers twice? Or will this overwrite the initial set?

Reply:
Maybe better way is to set like this https://www.adamsmith.haus/python/answers/how-to-set-response-headers-using-flask-in-python

Reply:
@shawaj It will override the existing headers. We should be able to uzse headers.append though